apps/api/src/campaign-application/campaign-application.service.ts

```diff
@@ -9,6 +9,13 @@ export class CampaignApplicationService {
   }
 
   create(createCampaignApplicationDto: CreateCampaignApplicationDto) {
+    if (
+      !createCampaignApplicationDto.acceptTermsAndConditions ||
+      !createCampaignApplicationDto.transparencyTermsAccepted ||
+      !createCampaignApplicationDto.personalInformationProcessingAccepted
+    ) {
+      throw new HttpException('All agreements must be checked', HttpStatus.METHOD_NOT_ALLOWED)
```

Comment:
`Method not allowed (405)` is not the appropriate http response to respond with in such scenarios. 
We've established some practice to respond with  `Bad Request (400)` for invalid DTO content, but I suppose you could also respond with `Unprocessable Content (422)`. Either of them are semantically fine.